app/controllers/notifications_controller.rb

```diff
@@ -1,6 +1,7 @@
 class NotificationsController < ApplicationController
   def create
-    NotificationWorker.perform_async(notification_params.to_json)
+    query_field = notification_params[:links].present? ? :links : :tags
```

Comment:
The links field could be present but not yet have been updated to contain topic links.  For example, there might be an entry in links for "available_translations", but no entry in topics (but some entries for topics under `tags`).  I think this needs to actually check if there are any matches in the links field, and add those to any matches based on the tags field.